### ISSUE TITLE
[0.17] ajv errors

### ISSUE THREAD

**Issue by yosa** [quasarframework/quasar#3198]
### Software version

perating System         	Linux(4.15.0-38-generic) - linux/x64
NodeJs                   	11.9.0

Global packages          	
  NPM                    	6.5.0
  yarn                   	Not installed
  quasar-cli             	0.17.23
  vue-cli                	3.4.0
  cordova                	Not installed

Important local packages 	
  quasar-cli             	0.17.23	(Quasar Framework CLI)
  quasar-framework       	0.17.19	(Build responsive SPA, SSR, PWA, Hybrid Mobile Apps and Electron apps, all simultaneously using the same codebase)
  quasar-extras          	2.0.9	(Quasar Framework fonts, icons and animations)
  vue                    	2.5.17	(Reactive, component-oriented view layer for modern web interfaces.)
  vue-router             	3.0.1	(Official router for Vue.js 2)
  vuex                   	3.0.1	(state management for Vue.js)
  electron               	Not installed
  electron-packager      	Not installed
  electron-builder       	Not installed
  @babel/core            	7.0.0-beta.54	(Babel compiler core.)
  webpack                	4.25.1	(Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.)
  webpack-dev-server     	3.1.14	(Serves a webpack app. Updates the browser on changes.)
  workbox-webpack-plugin 	3.6.3	(A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.)
  register-service-worker	1.5.2	(Script for registering service worker, with hooks)
Quasar:
@quasar/app (v1+ only):
OS:
Node:
NPM:
Browsers:
iOS:
Android:
Any other software related to your bug:

### What did you get as the error?
UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:19141) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

### What were you expecting?

That the quasar dev command start

### What steps did you take, to get the error?

1. Zero installation
```bash
npm install -g quasar-cli @vue/cli @vue/cli-init
quasar init test
````
1. Run quasar-dev
```bash
cd test
quasar dev
```

**Comment by rstoenescu** [comment]
Duplicate of Unhandled Promise Rejection Warning with cli 0.17.22 Running "quasar dev" w/ brand new install #209

**Comment by rstoenescu** [comment]
Turns out it's a problem with "ajv" package. For a temporary fix (until the owner of ajv releases a patch), add this to your `package.json`:

```
"resolutions": {
    "ajv": "6.8.1"
  },
```

**Comment by rstoenescu** [comment]
Reference: https://github.com/epoberezkin/ajv-errors/issues/26

**Comment by rstoenescu** [comment]
Pushed a temporary fix for 0.17 starter kit. Please try now @yosa 
I do recommend you try the v1.0 beta starter kit instead (https://v1.quasar-framework.org).

**Comment by yosa** [comment]
Thanks, it worked.
